dashboard/views.py

```diff
@@ -107,7 +92,7 @@ def file_access_within_week(request, course_id=0):
         "grade": grade,
         "course_id": course_id
     }
-    eventlog(request.user, EVENT_VIEW_FILE_ACCESS, None, json.dumps(data))
+    eventlog(request.user, EVENT_VIEW_FILE_ACCESS, extra=data)
```

Comment:
This is very elegant way of storing JSON data into extra field!

Reply:
Yeah, seemed super straightforward. I compared what's written out to the column and it's better. Looked like it was being double escaped before and not usable in tableau. Will just merge this.